config/config.go

```diff
@@ -79,3 +77,14 @@ func Load(logger logrus.FieldLogger) (Options, error) {
 
 	return cfg, nil
 }
+
+func SetDatabaseConfig(scheme string, url string, logger logrus.FieldLogger) (Options, error) {
+
+	viper.Set("db.scheme", scheme)
+	viper.Set("db.url", url)
+
+	viper.WriteConfig() 
```

Comment:
no need to write it. we prefer actually to use the env variables only. The file is there only for development reasons mostly.